opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/kafka-graphs.properties

```diff
@@ -1,52 +1,583 @@
+ AREA:value#edd400 \
+ LINE2:value#c4a000:"Leaders" \
+ GPRINT:value:AVERAGE:" Avg \\: %8.2lf %s" \
+ GPRINT:value:MIN:" Min \\: %8.2lf %s" \
+ GPRINT:value:MAX:" Max \\: %8.2lf %s\\n"
+
+
+# TODO: Combine these next two graphs?
+report.kafka.isrShrinksPerSec.name=In-Sync Replica Shrinks Per Second
+report.kafka.isrShrinksPerSec.columns=isrShrinksPerSec
+report.kafka.isrShrinksPerSec.type=interfaceSnmp
+report.kafka.isrShrinksPerSec.command=--title="In-Sync Replica Shrinks Per Second" \
+ --vertical-label="Shrinks per second" \
+ DEF:value={rrd1}:isrShrinksPerSec:AVERAGE \
+ AREA:value#edd400 \
+ LINE2:value#c4a000:"Shrinks per second" \
+ GPRINT:value:AVERAGE:" Avg \\: %8.2lf %s" \
+ GPRINT:value:MIN:" Min \\: %8.2lf %s" \
+ GPRINT:value:MAX:" Max \\: %8.2lf %s\\n"
+
+report.kafka.isrExpandsPerSec.name=In-Sync Replica Expansions Per Second
+report.kafka.isrExpandsPerSec.columns=isrExpandsPerSec
+report.kafka.isrExpandsPerSec.type=interfaceSnmp
+report.kafka.isrExpandsPerSec.command=--title="In-Sync Replica Expansions Per Second" \
+ --vertical-label="Expansions per second" \
+ DEF:value={rrd1}:isrExpandsPerSec:AVERAGE \
+ AREA:value#edd400 \
+ LINE2:value#c4a000:"Expansions per second" \
+ GPRINT:value:AVERAGE:" Avg \\: %8.2lf %s" \
+ GPRINT:value:MIN:" Min \\: %8.2lf %s" \
+ GPRINT:value:MAX:" Max \\: %8.2lf %s\\n"
+
+
+report.kafka.replicaMaxLag.name=Maximum Lag Between Replicas
+report.kafka.replicaMaxLag.columns=replicaMaxLag
+report.kafka.replicaMaxLag.type=interfaceSnmp
+report.kafka.replicaMaxLag.command=--title="Maximum Lag Between Replicas" \
+ --vertical-label="Messages" \
+ DEF:value={rrd1}:replicaMaxLag:AVERAGE \
+ AREA:value#edd400 \
+ LINE2:value#c4a000:"Maximum Lag" \
+ GPRINT:value:AVERAGE:" Avg \\: %8.2lf %s" \
+ GPRINT:value:MIN:" Min \\: %8.2lf %s" \
+ GPRINT:value:MAX:" Max \\: %8.2lf %s\\n"
+
+
+# TODO: Combine these next two graphs?
+report.kafka.purgatoryProduce.name=Purgatory Size: Produce
+report.kafka.purgatoryProduce.columns=purgatoryProduce
+report.kafka.purgatoryProduce.type=interfaceSnmp
+report.kafka.purgatoryProduce.command=--title="Purgatory Size: Produce" \
+ --vertical-label="Requests" \
+ DEF:value={rrd1}:purgatoryProduce:AVERAGE \
+ AREA:value#edd400 \
+ LINE2:value#c4a000:"Producer requests" \
+ GPRINT:value:AVERAGE:" Avg \\: %8.2lf %s" \
+ GPRINT:value:MIN:" Min \\: %8.2lf %s" \
+ GPRINT:value:MAX:" Max \\: %8.2lf %s\\n"
+
+report.kafka.purgatoryFetch.name=Purgatory Size: Fetch
+report.kafka.purgatoryFetch.columns=purgatoryFetch
+report.kafka.purgatoryFetch.type=interfaceSnmp
+report.kafka.purgatoryFetch.command=--title="Purgatory Size: Fetch" \
+ --vertical-label="Requests" \
+ DEF:value={rrd1}:purgatoryFetch:AVERAGE \
+ AREA:value#edd400 \
+ LINE2:value#c4a000:"Fetch requests" \
+ GPRINT:value:AVERAGE:" Avg \\: %8.2lf %s" \
+ GPRINT:value:MIN:" Min \\: %8.2lf %s" \
+ GPRINT:value:MAX:" Max \\: %8.2lf %s\\n"
+
+
+report.kafka.netProcAvgIdle.name=Network Processor Average Idle Percentage
```

Comment:
I'm going to change this... who graphs an idle percentage? I'm going to invert it so that it's a usage percentage. That makes 100% more sense.